examples/jigsaw-example/app/src/main/resources/de/saxsys/mvvmfx/examples/jigsaw/app/AppView.fxml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.layout.VBox?>
+<VBox xmlns="http://javafx.com/javafx"
+      fx:id="appMainBox"
+      xmlns:fx="http://javafx.com/fxml"
+      fx:controller="de.saxsys.mvvmfx.examples.jigsaw.app.AppView"
+      prefHeight="400.0" prefWidth="600.0">
+    <fx:include source="../../../../../../../../../../../../../../Projekte/mvvmFX/examples/jigsaw-example/circle/src/main/resources/de/saxsys/mvvmfx/examples/jigsaw/circle/CircleView.fxml" fx:id="circle" resources="circle" />
```

Comment:
This path is relative to your local file system. This won't work on other peoples machines.